parsers/nmparser/parse_run_details.go

```diff
@@ -31,27 +49,62 @@ func ParseRunDetails(lines []string) RunDetails {
 	covarianceTime := 0.0
 	functionEvaluations := int64(0)
 	significantDigits := 0.0
-	for _, line := range lines {
+	problemText := ""
+	modFile := "" // TODO, pass in from caller
+	estimationMethod := []string{}
+	dataSet := ""
+	numberOfPatients := int64(0)
+	numberOfObs := int64(0)
+	numberOfDataRecords := int64(0)
+	outputTable := ""
+
+	for i, line := range lines {
 		switch {
 		case strings.Contains(line, "1NONLINEAR MIXED EFFECTS MODEL PROGRAM (NONMEM) VERSION"):
 			nmversion = parseNMVersion(line)
 		case strings.Contains(line, "NO. OF FUNCTION EVALUATIONS USED"):
 			functionEvaluations, _ = strconv.ParseInt(replaceTrim(line, "NO. OF FUNCTION EVALUATIONS USED:"), 10, 64)
 		case strings.Contains(line, "NO. OF SIG. DIGITS IN FINAL EST.:"):
 			significantDigits, _ = strconv.ParseFloat(replaceTrim(line, "NO. OF SIG. DIGITS IN FINAL EST.:"), 64)
-		case strings.Contains(line, "Elapsed estimation time in seconds:"):
+		case strings.Contains(line, "Elapsed estimation"):
 			estimationTime = parseFinalTime(line)
 		case strings.Contains(line, "Elapsed covariance time in seconds:"):
 			covarianceTime = parseFinalTime(line)
+		case strings.Contains(line, "Elapsed postprocess time in seconds:"):
+			covarianceTime = parseFinalTime(line)
 		case strings.Contains(line, "Started"):
 			runStart = replaceTrim(line, "Started")
 		case strings.Contains(line, "Finished"):
 			runEnd = replaceTrim(line, "Finished")
+		case strings.Contains(line, "Stop Time:"):
+			if i+1 < len(lines) {
+				runEnd = lines[i+1]
+			}
+		case strings.Contains(line, "$PROB"):
+			problemText = replaceTrim(line, "$PROB")
+		case strings.Contains(line, "$ESTIMATION MAXEVAL"):
+			estimationMethod = append(estimationMethod, parseValue(line, "METH="))
+		case strings.Contains(line, "$DATA"):
+			dataSet = parseLine(line, 1)
+		case strings.Contains(line, "TOT. NO. OF INDIVIDUALS:"):
+			numberOfPatients, _ = strconv.ParseInt(replaceTrim(line, "TOT. NO. OF INDIVIDUALS:"), 10, 64)
+		case strings.Contains(line, "TOT. NO. OF OBS RECS:"):
+			numberOfObs, _ = strconv.ParseInt(replaceTrim(line, "TOT. NO. OF OBS RECS:"), 10, 64)
+		case strings.Contains(line, "NO. OF DATA RECS IN DATA SET:"):
+			numberOfDataRecords, _ = strconv.ParseInt(replaceTrim(line, "NO. OF DATA RECS IN DATA SET:"), 10, 64)
+		case strings.Contains(line, "$TABLE NOPRINT ONEHEADER FILE="):
```

Comment:
same thing as $EST, this is definitely too specific, we should just grab the entire table line, then parse it elsewhere